script.radioparadise/resources/skins/Default/720p/radioparadise.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
please rename to script-radioparadise.xml (we use this as a naming convention to avoid confusion with skin xml files)